### DISCUSSION TITLE
New window mode: "middle" third and fourth

### DISCUSSION THREAD

**Discussion by jpuffsh** [rxhanson/Rectangle#1261] [Ideas]
To fit 3 windows on 1 small-ish display (while also making them big enough to reasonably use), I'd want to be able to put one of the windows in the middle with the same size as the currently existing "first/last two thirds", and/or "first/last three fourths" modes. The other 2 windows would be thirds or fourths on the left and right, but that the middle third/fourth won't be maximized, so it won't completely cover the other 2 windows on the side. Meaning that I can quickly click them to put them in focus (in my use case at least).

**Comment by jpuffsh** [comment]
I'd want this to be in the same position, but bigger than the currently existing "center half" and "center third" modes.

**Comment by rxhanson** [comment]
If I understand correctly, I think you can accomplish what you're looking for by repeating the shortcut for first third or fourth to cycle through the different positions.

**Comment by jpuffsh** [comment]
With default settings:

https://github.com/rxhanson/Rectangle/assets/55923092/9cd5f459-2edf-42f4-9a5f-e21372cf1479

**Reply by jpuffsh** [reply]
This only snaps the window to the furthermost left or right of the screen. My aim is to have it snap to the middle, where the window size is the same, but has equal empty spacing on the left and right.

**Comment by jpuffsh** [comment]
![CleanShot 2023-09-20 at 21 11 23@2x](https://github.com/rxhanson/Rectangle/assets/55923092/f93b5e97-e3d4-4ce4-af77-1e0bca3672a3)

Just found this. Raycast's "center two thirds" and "center three fourths" will suffice for now.

**Comment by rxhanson** [comment]
If you were just looking for an additional custom size, centered command in Rectangle then you can configure one with a terminal command:

https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#add-an-extra-centering-command-with-custom-size

Or you can change the "almost maximize" size:

https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#almost-maximize

On the Rectangle UI, the app is purposefully slim.

Beyond that, [Rectangle Pro](https://rectangleapp.com/pro) does have more stock shortcuts and a custom shortcut builder and you could get any size/position you wanted.
